engines/bastion_katello/app/assets/javascripts/bastion_katello/capsule-content/capsule-content.factory.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @ngdoc service
+ * @name  Bastion.capsule-content.factory:CapsuleContent
+ *
+ * @requires BastionResource
+ *
+ * @description
+ *   Provides a BastionResource for capsule content.
+ */
+angular.module('Bastion.capsule-content').factory('CapsuleContent',
+    ['BastionResource', function (BastionResource) {
+
+        return BastionResource('/katello/api/capsules/:id/content/:action', {id: '@id'}, {
+          sync_status: {method: 'GET', isArray: false, params: {action: 'sync'}},
+          sync: {method: 'post', isArray: false, params: {action: 'sync'}},
+          cancel_sync: {method: 'delete', isArray: false, params: {action: 'sync'}}
```

Comment:
Switch the snake_case to camel_case or the linter will ding you. This is because most JavaScript is by nature camelCase oriented so we chose to write our code this way. API data from the server comes in snake_case, so we use [''] syntax to refrence it and find it helps to identify server vs. code data.